packages/shared_preferences/shared_preferences_foundation/darwin/Classes/BetterSharedPreferencesPlugin.swift

```diff
@@ -0,0 +1,131 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
```

Comment:
I'd call this `CachelessSharedPreferencePlugin`. it's hard to tell what`Better` means